sources/startup.c

```diff
@@ -801,6 +801,16 @@ classic:;
 	while ( *t ) { *s++ = *t++; i++; }
 	s[-2] = 'o'; *s = 0;
 	}
+/*
+	Try to create the sort file already, so we can Terminate earlier if this fails.
+*/
+	if ( ( AM.S0->file.handle = CreateFile((char *)AM.S0->file.name) ) < 0 ) {
+		MesPrint("Could not create sort file: %s", AM.S0->file.name);
+		Terminate(-1);
+	};
+	/* Reset the file.handle and clean up the test file */
+	AM.S0->file.handle = -1;
+	remove(AM.S0->file.name);
```

Comment:
Is it intentional not to close the file here?

Reply:
No, this was not intentional. I will fix it.